cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/UnmanagedClusterWizardAdvanced.tsx

```diff
@@ -0,0 +1,22 @@
+// React imports
+import React, { useContext } from 'react';
+
+// App imports
+import { UmcStore } from '../../../state-management/stores/Store.umc';
+import { UMC_ADVANCED_TAB_NAMES } from '../../../shared/constants/NavRoutes.constants';
+import Wizard from '../../../shared/components/wizard/Wizard';
+import UnmanagedClusterSettingsAdvanced from './unmanaged-cluster-settings-step/UnmanagedClustersSettingsAdvanced';
+import UnmanagedClusterNetworkSettings from './unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings';
+
+function UnmanagedClusterWizardAdvanced() {
+    const { umcState, umcDispatch } = useContext(UmcStore);
+
+    return (
+        <Wizard tabNames={UMC_ADVANCED_TAB_NAMES} state={umcState} dispatch={umcDispatch}>
```

Comment:
Nice! I'm liking how this turned out with the two different Wizards and the re-use of the UnmanagedClusterNetworkSettings.

Reply:
Thank you!